M2/BUILD/docker/README.md

```diff
@@ -0,0 +1,28 @@
+# Running Macaulay2 in Docker
+
+Docker is a software for building, shipping, or running applications on different
```

Comment:
Change "a software" to "software".

Reply:
I'll fix this in the next commit. Just need a few more things before I push.